commit 6605c7445a20c760b577842f8d1e5a95a0dccd1a (refs/changes/70/3770/1)
Author: Mike Zeller <mike.zeller@joyent.com>
Date:   2018-04-03T19:45:55-07:00 (1 year, 6 months ago)
    
    TRITON-305 TRITON-262 should be more careful with async.waterfull arguments

diff --git a/agent.js b/agent.js
index 96609ff..0efe864 100644
--- a/agent.js
+++ b/agent.js
@@ -275,7 +275,9 @@ function setInZoneAutoMetadata(callback) {
 				});
 			}
 		]
-	}, callback);
+	}, function doneInZoneAutoMetadata(err) {
+		callback(err);
+	});
 }
 
 async.waterfall([
